tests/e2e/data/config-taquito-test-environment-invalid-initial-storage-string.json

```diff
@@ -32,33 +32,33 @@
         }
     },
     "network": {
-        "ithacanet": {
-            "label": "ithacanet",
-            "rpcUrl": "https://ithacanet.ecadinfra.com",
-            "protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
+        "ghostnet": {
+            "label": "ghostnet",
+            "rpcUrl": "https://rpc.ithacanet.teztnets.xyz",
+            "protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
```

Comment:
This is fine for now but given that ghostnet will not have a stable protocol, we may want to consider how we can dynamically handle protocol upgrades on ghostnet moving forward

Reply:
Yeah we can query teztnets to see what is on the about page for ghostnet and just take the highest block protocol. The thing is these files are JSON and we may have to build these files if we want anything like that. That isn't a trivial thing to do so I am not going to do it now/here